app/course/controllers/CourseCtrl.js

```diff
@@ -274,6 +274,25 @@ courseApp.controller('CourseCtrl', ['$scope', '$rootScope', '$routeParams', '$ti
 
 		$scope.updateSection = function (section) {
 			courseActionCreators.updateSection(section);
+
+			if (isNumber(section.sequenceNumber) == false) {
```

Comment:
While guard clauses like these are generally good, in this case the behavior of updateSection() really is bifurcated depending on whether the sequenceNumber is a number, so I think wrapping it in an 'if' might be clearer.